src/lib/study.js

```diff
@@ -149,6 +149,11 @@ const anomalyGroups = [
         guidance: 'See the [`[Exposed]`](https://webidl.spec.whatwg.org/#Exposed) extended attribute section in Web IDL for requirements.'
       },
       { name: 'invalid', title: 'Invalid Web IDL' },
+      {
+        name: 'noEvent',
+        title: 'Suspicious event handlers',
```

Comment:
```suggestion
        title: 'Event handlers needing review',
```
not great either, but "suspicious" feels a bit strong

Reply:
I agree "suspicious" isn't good. Looking at the title of this PR, I'm thinking "Event handlers with no matching events" could be a good enough title in practice.